examples/server.mjs

```diff
@@ -0,0 +1,73 @@
+import fetch from "node-fetch";
```

Comment:
I think we should stick with standard `.js` files instead of explicitly specifying them as `.mjs` or `.cjs`. Refer to earlier comment about globally changing node to work in ES6 module mode isntead.
Ref: <https://stackoverflow.com/a/57492606>

Reply:
https://github.com/node-fetch/node-fetch#commonjs `node-fetch from v3 is an ESM-only module - you are not able to import it with require().` Is mostly the reason here.

Reply:
I can do the type module in package.json instead though.

Reply:
`.mjs` is easier to see from one file that is working this way rather than checking multiple places.